src/lib/cpu_info/cpu_info_funcs.c

```diff
@@ -0,0 +1,11 @@
+#include <lib/libcpu_info.h>
+#include <string.h>
+#include <assert.h>
+
+extern void set_feature_val(struct cpu_info *info, const char *name, unsigned int val) {
```

Comment:
No need to write `extern` if you define function here.

Reply:
Right, Thanks!